pkg/loader/compose/compose.go

```diff
@@ -273,12 +322,126 @@ func loadPorts(composePorts []string) ([]kobject.Ports, error) {
 	return ports, nil
 }
 
-// LoadFile loads compose file into KomposeObject
+// LoadFile loads a compose file into KomposeObject
 func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
+
+	// What will finally be returned
 	komposeObject := kobject.KomposeObject{
 		ServiceConfigs: make(map[string]kobject.ServiceConfig),
 		LoadedFrom:     "compose",
 	}
+
+	// Load the json / yaml file in order to get the version value
+	version, err := getVersionFromFile(files[0])
```

Comment:
And if this PR needs to go in on priority, then we need a high priority issue tracking this, WDYT?